.github/workflows/main.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         id: cache
         with:
           path: ./build/release/third_party/install
-          key: ${{ github.job }}-${{ hashFiles('./third_party/**') }}
+          key: ${{ github.job.name }}-${{ hashFiles('./third_party/**') }}
```

Comment:
Good catch! Maybe worth putting in a separate PR and merge it right away?

Reply:
https://github.com/mavlink/MAVSDK/pull/1814